portals/publisher/src/main/webapp/source/src/app/components/Apis/Details/Policies/Policies.tsx

```diff
@@ -74,14 +94,17 @@ const Policies: React.FC = () => {
     const [allPolicies, setAllPolicies] = useState<PolicySpec[] | null>(null);
     const [expandedResource, setExpandedResource] = useState<string | null>(null);
     const [isChoreoConnectEnabled, setIsChoreoConnectEnabled] = useState(api.gatewayType === 'wso2/choreo-connect');
+    const [isAPILevelTabSelected, setIsAPILevelTabSelected] = useState(api.apiPolicies != null,);
```

Comment:
Do we need the trailing comma here?

```suggestion
    const [isAPILevelTabSelected, setIsAPILevelTabSelected] = useState(api.apiPolicies != null);
```